thoth/storages/graph/postgres.py

```diff
@@ -4164,11 +4164,12 @@ def sync_analysis_result(self, document: dict) -> None:
         environment_name = document["metadata"]["arguments"]["extract-image"]["image"]
         os_name = document["result"]["operating-system"]["name"]
         os_version = document["result"]["operating-system"]["version_id"]
-        cuda_version = document["result"]["cuda-version"]["nvcc_version"]
-        if cuda_version != document["result"]["cuda-version"]["/usr/local/cuda/version.txt"]:
+        cuda_version = document["result"].get("cuda-version", {}).get("nvcc_version", None)
+        if cuda_version != document["result"].get("cuda-version", {}).get("/usr/local/cuda/version.txt", None):
             raise CudaVersionDoesNotMatch(
                 f"Cuda version detected by nvcc {cuda_version!r} is different from the one found in "
-                f"/usr/local/cuda/version.txt {document["result"]["cuda-version"]["/usr/local/cuda/version.txt"]!r}"
+                f"/usr/local/cuda/version.txt "
+                f"{document["result"].get("cuda-version", {}).get("/usr/local/cuda/version.txt", None)!r}"
```

Comment:
@fridex do you see any syntax error here?

Reply:
```
f"/usr/local/cuda/version.txt {document["result"]["cuda-version"]["/usr/local/cuda/version.txt"]!r}"
f"/usr/local/cuda/version.txt {document['result']['cuda-version']['/usr/local/cuda/version.txt']!r}"
```
I had to change the first to second to be able to run code in storages

Reply:
`" -> '`, :smile: , I did not suspect it matters, thanks @pacospace !

Reply:
Yes, now I see because we are using `f""` python didn't understand whether `"` means closing or string, so it is safe to use `'` inside `"`. Very good

Reply:
Thanks :)